modules/administration-guide/partials/proc_adding-or-removing-vscode-extensions-in-the-embedded-open-vsx-registry-instance.adoc

```diff
@@ -0,0 +1,83 @@
+:_content-type: PROCEDURE
+
+[id="adding-or-removing-vscode-extensions-in-the-embedded-open-vsx-registry-instance"]
+= Adding or removing pass:[<!-- vale RedHat.Spelling = NO -->] pass:[<!-- vale RedHat.Spelling = NO -->] VS Code pass:[<!-- vale RedHat.Spelling = YES -->] pass:[<!-- vale RedHat.Spelling = YES -->] extensions in the embedded Open VSX registry instance
+
+You can add or remove your choice of pass:[<!-- vale RedHat.Spelling = NO -->] pass:[<!-- vale RedHat.Spelling = NO -->] VS Code pass:[<!-- vale RedHat.Spelling = YES -->] pass:[<!-- vale RedHat.Spelling = YES -->] extensions in the embedded Open VSX registry instance that is cached by {prod-short} to support air-gapped, offline, and proxy-restricted environments.
+
+This will create a custom build of the Open VSX registry, which can be used in your organization’s workspaces. 
+
+NOTE: A subscription update reverts your `CheCluster` to default settings, and you must reapply your changes to `CheCluster`. 
```

Comment:
```suggestion
NOTE: Subscription updates for the {prod-short} operator will revert any customizations done to the `CheCluster` to the published settings for the updated operator; you must reapply your changes to the `CheCluster` after the update is complete.  You may also want to build a new registry container based on the updated {prod-short} container. 
```

Reply:
@nickboldt, I'd like to tweak your suggestion. Is the following okay?
 
```suggestion
WARNING: Subscription updates for the {prod-short} Operator revert any customizations of the `CheCluster` custom resource to the published settings for the updated operator. After the update is complete, you must reapply your changes to the `CheCluster` custom resource. You may also choose to build a new registry container based on the updated {prod-short} container, in order to get the latest security fixes and updates.
```